wiki/Beatmap_Spotlights/zh.md

```diff
@@ -0,0 +1,350 @@
+
+### osu!mania
+
+- **Playlist A**
+   - [An - ExaVid (Abraxos) \[at\_the\_threshold\]](https://osu.ppy.sh/beatmapsets/649306#mania/1589629)
+   - [Camellia - Artificial Snow (Dergo) \[ash's hard\]](https://osu.ppy.sh/beatmapsets/1026319#mania/2147852)
+   - [Culprate - Finger VIP (Parachor) \[Trigger Finger\]](https://osu.ppy.sh/beatmapsets/508572#mania/1081560)
+   - [Chroma - lost memory (Youwhang) \[a light received from you.\]](https://osu.ppy.sh/beatmapsets/961321#mania/2012530)
+   - [bbn - Identic Conflict (pporse) \[Birdcage\]](https://osu.ppy.sh/beatmapsets/246853#mania/568463)
+- **Playlist B**
+   - [X&G - Whiplash ft. josh pan (sakuraburst Remix) (error_exe777) \[break out\]](https://osu.ppy.sh/beatmapsets/723633#mania/1527737)
+   - [Igorrr & Ruby My Dear - Figue Folle (Parachor) \[Grotesque [Lv.18]\]](https://osu.ppy.sh/beatmapsets/347453#mania/782953)
+   - [Frums - theyaremanycolors (Vortex-) \[780THz\]](https://osu.ppy.sh/beatmapsets/829383#mania/1737654)
+   - [Silentroom - NULCTRL EX (-mint-) \[MOTION\]](https://osu.ppy.sh/beatmapsets/1086666)
+   - [Middleisland - Achromat (iJinjin) \[7K Black Another\]](https://osu.ppy.sh/beatmapsets/195940#mania/553820)
+- **Playlist C**
+   - [Synergy Sound - Dreaming (-MysticEyes) \[serenity\]](https://osu.ppy.sh/beatmapsets/966717#mania/2023410)
+   - [Azu - Haryu (asd123iii) \[Another\]](https://osu.ppy.sh/beatmapsets/183478#mania/524443)
+   - [KOAN Sound & Asa - fuego (sakuraburst remix) (Couil) \[MX\]](https://osu.ppy.sh/beatmapsets/622946#mania/1313121)
+   - [Camellia - POWA OF DA WILDANES (Monheim) \[YATSUKAMIZUOMITSUNO\]](https://osu.ppy.sh/beatmapsets/1098578#mania/2295244)
+   - [Camellia - Beyond the Geostationary Orbit Level (Leniane) \[Equatorial\]](https://osu.ppy.sh/beatmapsets/1011337#mania/2116795)
+- **Playlist D**
+   - [The Flashbulb - Every Two Days (Monheim) \[2\]](https://osu.ppy.sh/beatmapsets/905599#mania/1890121)
+   - [siqlo - One Way Street (Dreamwalker) \[Unilateral\]](https://osu.ppy.sh/beatmapsets/688453#mania/1867902)
+   - [Ta9-5 - Positive (feat. Looci) (Dubstek) \[Loop Loop Loop\]](https://osu.ppy.sh/beatmapsets/1026168#mania/2146057)
+   - [Camellia - PLANET//SHAPER (Evening) \[GEOLOGICAL//IRRESOLUTION\]](https://osu.ppy.sh/beatmapsets/376293#mania/823842)
+   - [Frums - We Want To Run (Blocko) \[Another\]](https://osu.ppy.sh/beatmapsets/1025170#mania/2335802)
+
+## 赛季
+
+月赛目前在规定时段中进行。每个赛季都包含一组图池和一个面向整个社区的竞争性赛季排行榜。
+
+1. 赛季持续 10 周。
+2. 赛季在开始前就已经做好了充足的准备。
+   - 图池都会在赛季开始之前被选中并锁定。
+   - 赛季开始后，发布整个时间表。
+3. 一个赛季被分为多个星期。每个星期都会被标上一个字母。
+   - 字母代表每星期的多人游戏房间的图池。
+   - 在整个赛季中，每周进行一次循环，并重复两次。最后两周将以前的所有图池分为两个合并的图池。
+4. 赛季结束后，在新赛季开始前将有三周休息时间。在此期间，可以对项目进行一些调整。
+
+### 时间表（Summer 2020）
+
+当前赛季的时移多人游戏大厅的可用时间。
+
+<!--The translation of "Timeshift" is a bit weird. Keep this note until the official translation is released.
+
+Note meanning: Timeshift Loobby is a multiplayer mode in lazer-->
+
+*小提示：时移多人游戏大厅（Timeshift Multiplayer Loobby）是 osu!lazer 中的一个多人游戏模式。*
```

Comment:
```suggestion
Note meanning: Timeshift Lobby is a multiplayer mode in lazer-->

*小提示：时移多人游戏大厅（Timeshift Multiplayer Lobby）是 osu!lazer 中的一个多人游戏模式。*
```
Was this a typo or what you actually meant?

Reply:
oh, sorry that actually typo